src/get-started/light-client.md

```diff
@@ -28,165 +28,272 @@ A transaction is an atomic operation that introduces changes to the blockchain
+  blockchain explorer of a node. Example:
 
     ```none
-    http://127.0.0.1:8200/api/services/timestamping/v1/timestamps
+    http://127.0.0.1:8200/api/explorer/v1/transactions
     ```
 
-- `explorerBasePath` represents API address of transaction explorer where
-  you can see transaction details at a blockchain node. Example:
-
-    ```none
-    http://127.0.0.1:8200/api/explorer/v1/timestamps/value?hash=
-    ```
-
-### Transfer Funds Transaction
+### Example of the Transfer Funds Transaction
 
 To execute this type of transaction you need to have two wallets
 created in advance: a sender and a receiver. You should also generate a separate
 key pair for each wallet, so that these keys could be used for defining and
 signing the transfer transactions between the wallets.
 
-Next, define `TransferFunds` transaction schema and data types as we did in the
-example above:
+Now, define `TransferFunds` transaction schema and data types as we did in the
+example above.
 
-```javascript
-const TransferFunds = Exonum.newMessage({
-  protocol_version: 0,
-  service_id: 128,
-  message_id: 0,
-  fields: [
-    { name: 'from', type: Exonum.PublicKey },
-    { name: 'to', type: Exonum.PublicKey },
-    { name: 'amount', type: Exonum.Uint64 },
-    { name: 'seed', type: Exonum.Uint64 }
-  ]
-})
+#### Define a Schema of the Transaction Payload with Protobuf
+
+In the transfer funds transaction we have only one entity - `TransferFunds` -
+that has two fields:
+
+- `amount` which represents the amount of transferred tokens
+- `seed` which represents a random number that guarantees non-idempotence of
+  transactions.
+
+!!! note
+    As you might know from our very first tutorial
+    [Cryptocurrency Tutorial](create-service.md),
+    in order to transfer funds from one wallet to another, a `seed` is included
+    into each such transaction. This prevents transactions from being hacked by
+    a third person. You can generate `seed` as follows:
+
+      ```javascript
+      const seed = Exonum.randomUint64()
+      ```
+
+Below is an example of the `example.proto` file describing the data schema of
+the transfer funds transaction in the Protobuf format:
+
+```proto
+package examples;
+
+message TransferFunds {
+  uint64 amount = 1;
+  // Auxiliary number to guarantee non-idempotence of transactions.
+  uint64 seed = 2;
+}
 ```
 
-As you might know from our very first tutorial
-[Cryptocurrency Tutorial](create-service.md),
-in order to transfer funds from one wallet to another, a `seed` is included
-into each such transaction. This prevents transactions from being hacked by
-a third person. You can generate `seed` as follows:
+As the `.proto` file is ready, generate the `*.js` file as follows:
+
+```json
+{
+  "proto": "pbjs --keep-case -t static-module example.proto -o ./proto.js"
+}
+```
+
+#### Define a Transaction Schema
+
+Define `TransferFunds` transaction schema and its field types:
 
```

Comment:
as soon as `TransferFunds` demonstrate using of proto stubs (`.js` files), please also show how to include them on the page before defining transaction:
```
import * as proto from 'stubs.js'
```